common/src/test/java/com/revenuecat/purchases/common/BackendTest.kt

```diff
@@ -74,8 +74,8 @@ class BackendTest {
     private val headersSlot = slot<Map<String, String>>()
```

Comment:
these are just formatting updates, i'm going to split unit tests to a separate PR